src/hooks/useExternalWalletSectionsData.ts

```diff
@@ -15,11 +15,12 @@ export default function useExternalWalletSectionsData({ address, type }: { addre
   const { data: hiddenTokens } = useFetchHiddenTokens({ address });
   const { data: showcaseTokens } = useFetchShowcaseTokens({ address });
 
-  const sellingTokens = useMemo(() => uniqueTokens?.filter(token => token.currentPrice) || [], [uniqueTokens]);
+  // TODO: Bring back selling tokens eventually
+  // const sellingTokens = useMemo(() => uniqueTokens?.filter(token => token.) || [], [uniqueTokens]);
```

Comment:
We do not have this data right now, so removing this and bringing it back later once we have a better nft vendor